entity-framework/core/managing-schemas/scaffolding.md

```diff
@@ -37,11 +37,12 @@ Scaffold-DbContext 'Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=Chinook'
 
 ### Configuration and User Secrets
 
-If you have an ASP.NET Core project, you can use the `Name=<connection-string>` syntax to read the connection string from configuration.
+If you have a .NET or ASP.NET Core project, you can use the `Name=<connection-string>` syntax to read the connection string from configuration.
```

Comment:
I'm not sure this is helpful. This syntax will only work with .NET applications that are using D.I. and configuration. Usually, this means only ASP.NET Core projects.

Reply:
> I'm not sure this is helpful. This syntax will only work with .NET applications that are using D.I. and configuration. Usually, this means only ASP.NET Core projects.

The configuration system is .NET, not just ASP.NET Core. See [this comment with a snippet](https://github.com/dotnet/EntityFramework.Docs/issues/3939#issuecomment-1191978026)

Reply:
Understood, but how many people use it outside of ASP.NET Core? Which other templates use it by default?

Reply:
> Understood, but how many people use it outside of ASP.NET Core? Which other templates use it by default?

Plenty of folks who are reverse engineering the DB once they know it works with .NET. As written, the implication is it doesn't work outside of ASP.NET Core and you can't use user secrets with .NET console apps, which is not correct.

Reply:
See [Dependency injection in .NET](https://docs.microsoft.com/en-us/dotnet/core/extensions/dependency-injection)

It's not just ASP.NET Core.

Reply:
How about, "If you have a .NET application that uses the hosting model and configuration system, such as an ASP.NET Core project, then you can use the `Name=<connection-string>` syntax to read the connection string from configuration."

Reply:
such as an ASP.NET Core or Worker Service project

Reply:
@ericstj @IEvangelist please review wording for using configuration with .NET

Reply:
I think @ajcvickers wording sounds OK.  Would it help to provide a link to hosting model and configuration system?  cc @dotnet/area-extensions-hosting @dotnet/area-extensions-configuration